lstchain/scripts/lstchain_mc_sensitivity.py

```diff
@@ -82,67 +90,61 @@ def main():
     gammas_mc = dl2[dl2.mc_type == 0]
     protons_mc = dl2[dl2.mc_type == 101]
     good_gammas = dl2
-
-    sns.distplot(gammas_mc.gammaness, label='gammas')
-    sns.distplot(protons_mc.gammaness, label='protons')
+    
+    sns.distplot(gammas_mc.gammaness, label='gammas');
+    sns.distplot(protons_mc.gammaness, label='protons');
     plt.legend()
     plt.tight_layout()
     plt.show()
-    sns.distplot(gammas_mc.mc_energy, label='gammas')
-    sns.distplot(protons_mc.mc_energy, label='protons')
+    sns.distplot(gammas_mc.mc_energy, label='gammas');
+    sns.distplot(protons_mc.mc_energy, label='protons');
     plt.legend()
     plt.tight_layout()
     plt.show()
-    sns.distplot(gammas_mc.log_reco_energy, label='gammas')
-    sns.distplot(protons_mc.log_reco_energy, label='protons')
-    plt.legend()
+    sns.distplot(gammas_mc.reco_energy, label='gammas');
+    sns.distplot(protons_mc.reco_energy, label='protons');
+    plt.legend();
     plt.tight_layout()
     plt.show()
     ctaplot.plot_theta2(gammas_mc.reco_alt, gammas_mc.reco_az, gammas_mc.mc_alt, gammas_mc.mc_az, range=(0, 1), bins=100)
     plt.show()
     plt.figure(figsize=(12, 8))
-    ctaplot.plot_angular_res_per_energy(gammas_mc.reco_alt,
-                                        gammas_mc.reco_az,
-                                        gammas_mc.mc_alt,
-                                        gammas_mc.mc_az,
-                                        gammas_mc.reco_energy,
-                                        )
-
+    ctaplot.plot_angular_res_per_energy(gammas_mc.reco_alt, gammas_mc.reco_az, gammas_mc.mc_alt, gammas_mc.mc_az,
```

Comment:
shall we include ctaplot in the install?
this function has changed in the latest release, so a version of ctaplot would be required and a test.

Reply:
comments below